src/limiter/limiter.go

```diff
@@ -142,7 +142,7 @@ func (l *Limiter) RequestSuccessful(requestWeight int) error {
 //the RateLimitConfig in the Limiter struct to prevent more 429s in the future.
 //The wait parameter is optional. It is the amount of time in milliseconds after HitRateLimit is
 //called that no requests will be allowed.
-func (l *Limiter) HitRateLimit(requestWeight int) error {
+func (l *Limiter) HitRateLimit(requestWeight int, wait ...int64) error {
```

Comment:
what's the thought around `...int64` here? Not seeing the use case so hard to follow

Reply:
the ...int64 makes it optional. The use case is that different APIs lock you out for different period of times after you hit the rate limit. For example I believe that kraken suspends your requests for fifteen minutes after hitting their rate limit.

Reply:
Ok. Mentioned this below but I feel like that belongs in the config. It’s not going to change after the config is set so it seems odd to pass it into this function versus referencing the config when the rate limit is hit.